.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
What do we need this file for?

Reply:
This is the standard project file from eclipse. So you can open Stellarium as eclipse project.